README.md

```diff
@@ -0,0 +1,228 @@
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+ Get your pc.
+ Clone this project to your local machine. 
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+Basic knownledge of computer.
+
+In order to run this project you need:
+live server.
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/EnixWord/MY_PORTFOLIO.git
+
+### Install
+
+Install this project with:
+cd to My Portfolio.
+
+### Usage
+
+To run the project, execute the following command:
+live server
+<!--
+Example command:
+
+```sh
+  
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+npm test
+
+### Deployment
+
+You can deploy this project using:
+github pages.
+
+<!-- AUTHORS -->
+
+## 👥 Author(s) <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@EnixWord](https://github.com/EnixWord)
+- Twitter: [@StevMicroverse](https://twitter.com/StevMicroverse)
+- LinkedIn: [Stephen Okhaishie](https://www.linkedin.com/in/stephen-okhaishie-3baa19255/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- **[future_feature_1]** In the future I will add work section.
+- **[future_feature_2]** I will add about section.
+- **[future_feature_3]** Contact section.
+- **[future_feature_4]** Desktop version design.
+- **[future_feature_5]** Refactoring the work section dynamically with Javascript.
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/EnixWord/MY_PORTFOLIO/issues).
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project give a ⭐️.
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank God Almighty and Microverse for the opportunity and motivation.
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+ My Portfolio Project
+
+> Description of the project.
+This is an HTML & CSS My Portfolio project
+
+## Built With
+- Major languages HTML, CSS
+
+[Live Demo Link](https://github.com/EnixWord/Hello_Microverse)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Web Browser
+Code Editor
+### Setup
+~~~bash
+git clone https://github.com/EnixWord/Hello_Microverse
+cd hello-world
+~~~
+
```

Comment:
- Hey, I think you've duplicated this part. Please, kindly double-check the readme file to fix this consistency issue regarding following the readme template structure to make your readme more professional.